tests/framework/InputTemplates/UQTemplate/uq_template_input.i

```diff
@@ -7,6 +7,6 @@ x = 100, 50
 y = 100, 50
 
 [settings]
-case = sample_mirrow
+case = sample_mirror
```

Comment:
This change appears to be creating problems; maybe it needs to be propagated or reverted?

Reply:
reverted...sorry.